### PR TITLE
Adjust to new package naming schema

### PR DESCRIPTION
**Pull request by dirkmueller**
We got rid of the barclamp- prefix.

### CODE REVIEW

**Comment by tboerger** [comment]
:+1:

**Comment by rsalevsky** [comment]
+1

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -792,8 +791,8 @@ for i in $required_barclamps ; do
 done
 
 # Install optional barclamps if they're present
-for i in updater suse-manager-client nfs_client \
-    cisco-ucs hyperv heat ceilometer manila trove tempest ; do
+for i in \
```
Comment (aspiers) [inline scripts/install-chef-suse.sh]: No need for the line-break any more ;-)

File: scripts/install-chef-suse.sh [hunk 2]
```diff
@@ -778,13 +778,12 @@ fi
 # Take care that the barclamps are installed in the right order (as expressed
 # in cookbook dependencies)
 #
-required_barclamps="crowbar deployer dns ipmi logging network ntp provisioner"
+required_barclamps="core"
```
Comment (aspiers) [inline scripts/install-chef-suse.sh]: IIUC, this variable name is no longer correct.  `core` is not a barclamp - it's a group of barclamps.  Right?  Same with `openstack`.

**Comment by rsalevsky** [comment]
I think the second commit can be  dropped anyway.

**Comment by aspiers** [comment]
Gate failure looks related to this change, or at least related to the repo merge.

**Comment by rsalevsky** [comment]
Superseded by #2146

**Pull request closed without merging**